apps/src/lib/levelbuilder/lesson-editor/ResourcesEditor.jsx

```diff
@@ -116,9 +119,33 @@ export default class ResourcesEditor extends Component {
     this.setState({resources});
   };
 
+  handleAddResourceClick = e => {
+    e.preventDefault();
+    this.setState({newResourceDialogOpen: true});
+  };
+
+  handleNewResourceDialogClose = () => {
+    this.setState({newResourceDialogOpen: false});
+  };
+
   render() {
     return (
       <div>
+        <AddResourceDialog
+          isOpen={this.state.newResourceDialogOpen}
+          onSave={this.addResource}
+          handleClose={this.handleNewResourceDialogClose}
+          typeOptions={[
```

Comment:
Can we just make this list of typeOptions a constant in the dialog or are we going to get this information from somewhere else in the future?

Reply:
making it a list of constants sounds right to me. you can always move it up/out if you start needing it elsewhere. it could even move into shared constants if we start needing the same list in ruby + js.

Reply:
Yeah, I played with having this being set in the ruby code at first then moved it here. I'll move it into the dialog as a constant.